authentication/models.py

```diff
@@ -238,3 +239,17 @@ class TwitterConnection(BaseThirdPartyConnection):
         max_length=255, unique=True, blank=True, null=True
     )
     driver = TwitterDriver()
+
+
+class ENSSaveError(Exception):
+    pass
+
+
+class ENSConnection(BaseThirdPartyConnection):
```

Comment:
you should override method is_connected

Reply:
why we should override is_connected?
it will return true if connection exists in constraints i should use it right?

Reply:
No, for example, in the case of ENS, you might transfer ENS to another address. So, every time a user calls the get connection API, we should check if the ENS is still verified or not.

Reply:
can a user have multiple ens connections?
if not i was thinking of something like this:
```
@classmethod
def is_connceted(cls):
    user_profile = cls.user_profile
    return cls.objects.filter(user_profile=user_profile).exists()
```

Reply:
or i should filter based on name of connection?
```
@classmethod
def is_connceted(cls):
    user_profile = cls.user_profile
    return cls.objects.filter(user_profile=user_profile, name=cls.name).exists()
```

Reply:
Please rebase your code with the new version of development. The function is_connected has changed.